.github/workflows/gen-docs.yml

```diff
@@ -0,0 +1,36 @@
+name: Generate Documentation
+
+on: [release]
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  build:
+
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
Maybe we want to name this one as well? "Checkout"?

Reply:
ok